scripts/entry

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+source $(dirname $0)/lib/debug_functions
+
+trap "chown -R $DAPPER_UID:$DAPPER_GID ." exit
+
+mkdir -p bin dist output
+
+# download all go mod modules to the vendor directory 
+if [[ ! -d vendor ]]; then
+	./scripts/vendor
```

Comment:
Wouldn't this fail?

Reply:
yeah b/c the vendor script is missing.  I created it locally but it didn't show up in git staging due to _vendor_ listed in the .gitignore 😄 I'll fix the .gitignore - nice catch.